aya/src/programs/utils.rs

```diff
@@ -0,0 +1,24 @@
+//! Common functions shared between multiple eBPF program types.
+use std::{ffi::CStr, os::unix::io::RawFd};
+
+use crate::{
+    programs::{FdLink, LinkRef, ProgramData, ProgramError},
+    sys::bpf_raw_tracepoint_open,
+};
+
+/// Attaches the program to a BTF id.
+pub(crate) fn attach_btf_id(
```

Comment:
I think this should be renamed to `attach_raw_tracepoint`. It doesn't take any BTF arguments, it *is* used to attach BTF programs, but what it really does is call BPF_RAW_TRACEPOINT_OPEN.

Reply:
Done